data/olm-catalog/ssp-operator.clusterserviceversion.yaml

```diff
@@ -454,6 +488,8 @@ spec:
                 - name: KVM_INFO_IMAGE
                 - name: VALIDATOR_IMAGE
                   value: quay.io/kubevirt/kubevirt-template-validator:latest
+                - name: VM_CONSOLE_PROXY_IMAGE
+                  value: quay.io/kubevirt/vm-console-proxy:v0.1.0
```

Comment:
Let's move this change into another PR, and just hardcode the version for now.
I don't want to make this PR more complex.